assets/stylesheets/main.scss

```diff
@@ -68,9 +68,9 @@
 @import '../pages/bundles-landing/bundlesLanding';
 @import '../pages/bundles-landing/support-landing-ab-test/supportLandingOld';
 @import '../pages/contributions-landing/contributionsLanding';
-@import '../pages/contributions-landing-uk/contributionsLandingUK';
-@import '../pages/contributions-landing-us/contributionsLandingUS';
-@import '../pages/contributions-landing-eu/contributionsLandingEU';
+@import '../pages/contributions-landing-new/contributionsLandingUK';
```

Comment:
Can we change contributions-landing-new to contributions-landing and get rid of the old one?

Reply:
Australia stills use the old one.

Reply:
We can't get rid of the old folder until the AU page has been migrated to Circles.

Reply:
In the meantime let's move the au one to contributions-landing-au and rename this one to contributions-landing

Reply:
Done.